kimchi/src/circuits/domain_constant_evaluation.rs

```diff
@@ -0,0 +1,79 @@
+use crate::circuits::domains::EvaluationDomains;
+use crate::circuits::polynomials::permutation::zk_polynomial;
+use crate::circuits::polynomials::permutation::ZK_ROWS;
+use ark_ff::FftField;
+use ark_poly::EvaluationDomain;
+use ark_poly::UVPolynomial;
+use ark_poly::{univariate::DensePolynomial as DP, Evaluations as E, Radix2EvaluationDomain as D};
+use serde::{Deserialize, Serialize};
+use serde_with::serde_as;
+
+use super::polynomials::permutation::vanishes_on_last_4_rows;
+// use super::zk_polynomial::ZkPolynomial;
+
+#[serde_as]
+#[derive(Clone, Serialize, Deserialize, Debug)]
+pub struct DomainConstantEvaluations<F: FftField> {
```

Comment:
add documentation here (for example, "pre-computed polynomials that depend only on the chosen field and domain")